### PR TITLE
Disable actions and events for amp-story[standalone]

### LINKED ISSUE #11855: Blacklist `on` attribute for <amp-story standalone>
The AMP actions/events system would allow developers to override performance and usability constraints that are baked into AMP story (e.g. playing offscreen videos, navigating to external URLs, submitting form data, etc.)

I think we should consider preventing this entirely; a naive way of doing so would be to blacklist the `on` attribute in validation.

Open to thoughts; /cc @choumx @dvoytenko @alanorozco @prateekbh

### PR DESCRIPTION
**Pull request by newmuis**
Fixes #11855.

### CODE REVIEW

**Comment by newmuis** [comment]
Ping @choumx

**Comment by newmuis** [comment]
Ping @choumx can you take a look?

File: src/service/action-impl.js [hunk 1]
```diff
@@ -646,6 +646,16 @@ export function parseActionMap(s, context) {
 
   let actionMap = null;
 
+  const isStandaloneStory = !!closest(context, el => {
+    return el.classList.contains('i-amphtml-story-standalone');
```
Comment (choumx) [inline src/service/action-impl.js]: Possible race condition since this requires `amp-story` extension to be downloaded and built.

**Comment by choumx** [comment]
...and sorry for letting this languish for 9 days. 😭

**Comment by newmuis** [comment]
I don't think it suffices to enforce at the `amp-story` level, since e.g. you could have a button outside of an `amp-story` that pauses a video inside the `amp-story`.  Also, all `amp-story[standalone]` stories take the mindset that the entire document "belongs to" the story.

That said, your suggestion SGTM, I'll investigate that implementation later in the week.

**Comment by newmuis** [comment]
@choumx Won't anything in `BaseElement` also block on the download of the extension JS, thereby causing a race condition?  Seems like we need to either: 

- Accept the race condition (in which case, I agree, a `BaseElement` solution seems cleaner)
- Bake this assumption into the runtime, looking _only_ at publisher-authored DOM

Is this correct?

**Comment by choumx** [comment]
There's an action queuing mechanism for extensions IIRC. Actually maybe let me take a stab at this on the airplane -- I'd like to extend/clean up the whitelisting mechanism and use that instead.

**Comment by newmuis** [comment]
Closed in favor of #14628.

**Pull request closed without merging**